src/api/ApiResponder.php

```diff
@@ -132,6 +178,13 @@ protected function get_interface_response_joinOpenGame($interface, $args) {
         return $success;
     }
 
+    /**
+     * Interface redirect for selectButton
+     *
+     * @param BMInterface $interface
+     * @param array $args
+     * @return bool
```

Comment:
Huh, is it actually the case that some of these interface functions return NULL and some return FALSE?  We might want to standardize that at some point if so.